tests/golden/check-cli/check-cli.bats

```diff
@@ -77,3 +77,10 @@ load '../helpers'
 
   rm /tmp/check-cli.test
 }
+
+@test "Single file as root" {
+  run xrefcheck \
+    --root single-file.md
+
+  assert_output --partial "Repository's root does not seem to be a directory: single-file.md"
```

Comment:
Let's:
* add `assert_failure` to check that the application exited with a non-zero status code.
* remove `--partial`, because we don't expect anything else to be printed to stdout

```suggestion
  assert_failure
  assert_output "Repository's root does not seem to be a directory: single-file.md"
```

Reply:
> remove --partial, because we don't expect anything else to be printed to stdout

Turns out that [`assert_output` captures stderr too](https://bats-core.readthedocs.io/en/stable/faq.html?highlight=assert_output#how-do-i-see-the-output-of-the-command-under-run-when-a-test-fails) :) 

So the output seems to contain 4 lines, including `Configuration file not found, using default config for GitHub repositories`